1-node-farm/starter/node_modules/.bin/is-ci

```diff
@@ -0,0 +1 @@
+../is-ci/bin.js
```

Comment:
Please add a .gitignore and remove node_modules from branch

Reply:
Added